src/plan/push.go

```diff
@@ -35,12 +35,12 @@ func parentExists(objectState model.ObjectState, currentState *state.State) bool
 		return true
 	case *model.ConfigState:
 		config := v.Remote
-		branch := currentState.GetBranch(*config.BranchKey(), false)
+		branch := currentState.Get(*config.BranchKey()).(*model.BranchState)
```

Comment:
`GetBranch, GetConfig` ... nahradene `Get` a `getOrCreate` (to `false` v starom kode je `create = false`).